gum/arch-x86/gumx86writer.c

```diff
@@ -236,8 +236,11 @@ gum_x86_writer_flush (GumX86Writer * self)
         if (self->target_cpu == GUM_CPU_AMD64)
           *((guint64 *) (r->address - 8)) = GUINT64_TO_LE (target_pc);
         else
+#if G_BYTE_ORDER == G_LITTLE_ENDIAN
           *((guint32 *) (r->address - 4)) = GUINT32_TO_LE (target_pc);
-
+#else
+          *((guint32 *) (r->address - 4)) = GUINT32_TO_BE (target_pc);
+#endif
```

Comment:
Let's add braces to both sides of this `if`, as this is a bit hard to read now.

Reply:
Ok. Think I got it.